### ISSUE TITLE
 which dataset did you use? could you provide the link?

### ISSUE THREAD

**Issue by YiChenCityU** [HorizonAD/stereo_dso#2]
which dataset did you use? could you provide the link?
 Thanks.

**Comment by JiatianWu** [comment]
I used Kitti Odometry dateset: http://www.cvlibs.net/datasets/kitti/eval_odometry.php

**Comment by marvis** [comment]
Hi @JiatianWu. Is there big difference between grayscale images and color images when running the results? Thanks!

**Comment by JiatianWu** [comment]
Not exactly. Stereo DSO transforms color image to grayscale.

**Comment by taiping-z** [comment]
Hi, @JiatianWu.

How to calc **disparity** in Calibration File for the KITTI datasets, could you please provide a simple example or a link?

Thanks!

**Comment by ouerkakaChango** [comment]
@taiping-z 
try this for  KITTI 00:
718.856	718.856	607.1928 185.2157 0
1241 376
crop
640 480
0.3861448
(The so called 'disparity' is baseline,I found the parameter in ORB-SLAM2's yaml,and divide 1000)

**Comment by JiatianWu** [comment]
Yes the disparity is actually baseline of the stereo camera. Already update it. Sorry for the ambiguity.

**Comment by taiping-z** [comment]
@ouerkakaChango, @JiatianWu,
Thanks a lot.

**Comment by ganleiboy** [comment]
@ouerkakaChango Thanks a lot. I have made camera.txt, but i don't kown why dividing 1000 is needed. Could you explain it in detail, please?
